clusterloader2/cmd/clusterloader.go

```diff
@@ -61,7 +60,7 @@ var (
 )
 
 func initClusterFlags() {
-	flags.StringEnvVar(&clusterLoaderConfig.ClusterConfig.KubeConfigPath, "kubeconfig", "KUBECONFIG", "", "Path to the kubeconfig file")
+	flags.StringEnvVar(&clusterLoaderConfig.ClusterConfig.KubeConfigPath, "kubeconfig", "KUBECONFIG", "", "Path to the kubeconfig file (optional, if empty, creates using in-cluster client config)")
```

Comment:
I'm fine with supporting incluster config, but I think this is a bit too implicit.

How about adding an explicit "run-from-cluster" (or sth along those lines) flag and:
- if true, create in-cluster config and validate kubeconfig is unset
- if false, use kubeconfig (and validate it is set)

?

Reply:
Good point. Will add that flag.